src/NServiceBus.Core.Tests/AssemblyScanner/AssemblyScannerTests.cs

```diff
@@ -267,6 +325,12 @@ public DynamicAssembly(string nameWithoutExtension, DynamicAssembly[] references
                     param.ReferencedAssemblies.Add(reference.FilePath);
                 }
 
+                if (executable)
```

Comment:
this code wouldn't be needed it would be enough to just have the "exe" file extension, but I feel this is more correct and not much extra effort. Thoughts?